eyes_core/applitools/core/server_connector.py

```diff
@@ -165,15 +229,21 @@ class ServerConnector(object):
 
     _is_session_started = False
 
-    def __init__(self):
-        # type: () -> None
+    def __init__(self, client_session=None):
+        # type: (ClientSession) -> None
```

Comment:
need to change type annotation to Optional[ClientSession]

Reply:
got it! changed in the following commit